library/src/test/java/org/apache/apex/malhar/lib/window/accumulation/PojoOuterJoinTest.java

```diff
@@ -277,4 +277,85 @@ public void PojoFullOuterJoinTest()
     Assert.assertEquals("Bob", testOutClass.getUName());
     Assert.assertEquals(0, testOutClass.getAge());
   }
+
+  @Test
+  public void PojoRightOuterJoinTestSeparateLeftAndRightKeys()
```

Comment:
No need to add a new test... Just modify the existing test to user the new constructor.